### DISCUSSION TITLE
UHD630核显开启硬解，请朋友们指点一下

### DISCUSSION THREAD

**Discussion by vbskycn** [monlor/docker-xiaoya#136] [Q&A]
请会的朋友指点一下

```
    devices:
      - /dev/dri:/dev/dri # Intel/AMD GPU
      # - /dev/nvidia0:/dev/nvidia0 # NVIDIA GPU 示例
      # - /dev/nvidiactl:/dev/nvidiactl
      # - /dev/nvidia-uvm:/dev/nvidia-uvm
    runtime: nvidia # 如果使用 NVIDIA GPU，启用此行   
```
是不是在docker-compose.yml的emby小节加上这几行代码就可以启用硬解了
![image](https://github.com/monlor/docker-xiaoya/assets/34542650/05fad561-bab3-442b-ba4c-bc99212d07b8)

**Comment by wangjh9712** [comment]
请参考 https://tech.dukeyin.com/2022/05/01/%E7%BE%A4%E6%99%96docker%E7%89%88emby%E5%BC%80%E5%90%AF%E7%A1%AC%E4%BB%B6%E8%A7%A3%E7%A0%81/

> 对于已经安装好了的Docker Emby，可以先停止容器，然后编辑容器，选择高级选项，在环境变量中修改 或 增加：

  ```
  GIDLIST = 0
  UID = 0  //使用root用户（通常ID为0）执行emby
  GID = 0  //使用默认显卡驱动（通常ID为0）解码
  device = /dev/dri:/dev/dri  //映射显卡驱动目录到Emby容器，并且指定为渲染设备
  ```
